scripts/deployments/UpgradeProxiedICS26Router.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.28;
+
+import { Deployments } from "../helpers/Deployments.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { TimelockController } from "@openzeppelin-contracts/governance/TimelockController.sol";
+import { ICS26Router } from "../../contracts/ICS26Router.sol";
+
+// solhint-disable custom-errors,gas-custom-errors
+
+library UpgradeProxiedICS26Router {
```

Comment:
Is this script tested anywhere?

Reply:
yes, it'll be, haven't pushed up the full e2e test with TimelockController yet